requirements.txt

```diff
@@ -3,6 +3,7 @@ Django==3.0.4
 django_extensions==2.2.2
 django-widget-tweaks==1.4.5
 icalendar==4.0.3
+ipdb==0.13.2
```

Comment:
ipdb is not a requirement. The common pattern is to add development dependencies in a `dev-dependencies.txt` file instead. Background: You don't want any unnecessary code on production.

Reply:
removed, will add dev-dependencies in another pr